libs/wingsdk/src/target-sim/dynamodb-table.inflight.ts

```diff
@@ -51,14 +52,16 @@ export class DynamodbTable
         containerPort: "8000",
       });
 
+      this._endpoint = `http://0.0.0.0:${hostPort}`;
```

Comment:
Why don't we use 127.0.0.1 or localhost here?

Reply:
I am not sure... That's how it's currently set up.